test/inductor/test_distributed_patterns.py

```diff
@@ -13,76 +12,100 @@
 
 # Fake distributed
 WORLD_SIZE = 2
-RESIZE = True
 
 
-def init_fake_distributed():
+def init_fake_distributed(device="cpu"):
     @torch.no_grad
     def all_gather(t):
-        return torch.cat([t] * WORLD_SIZE, 0)
+        # clone since all_gather input and output should not be aliases.
+        return torch.cat([t] * WORLD_SIZE, 0).clone()
```

Comment:
I think `cat()` will always create a copy`? (I doubted that for a second if there is one argument to cat, but i think this is definitely true when `WORLD_SIZE==2`).

Reply:
yes you are right - removed clone()